package.json

```diff
@@ -4,14 +4,52 @@
   "version": "0.0.1",
   "description": "The addon where ideas come to idea",
   "main": "index.js",
-  "author": "Les Orchard <lorchard@mozilla.com>",
+  "author": [
+    "Les Orchard <lorchard@mozilla.com>",
+    "Dave Justice <djustice@mozilla.com"
+  ],
   "engines": {
     "firefox": ">=38.0a1",
     "fennec": ">=38.0a1"
   },
-  "license": "MIT",
+  "scripts": {
+    "start": "st -nc .",
+    "package": "jpm xpi && mv @idea-town-addon-0.0.1.xpi dist/idea-town-addon-0.0.1.xpi"
+  },
+  "license": "MPL-2.0",
+  "dependencies": {
+    "exclude": "^1.0.0",
+    "xtend": "^4.0.0"
+  },
   "devDependencies": {
-    "gulp": "^3.9.0",
-    "gulp-connect": "^2.2.0"
+    "st": "^0.5.5"
+  },
+  "CONFIG": {
+    "CONTENT_URL": "http://localhost:1337/stub-content/index.html",
+    "BADGE_COLOR": "#00AAAA",
+    "ALLOWED_ORIGINS": [
+      "http://localhost:1337/*",
+      "http://127.0.0.1:1337/*",
+      "https://ideatown.firefox.com/*",
+      "https://mozilla.github.io/idea-town-addon/stub-content/",
+      "https://people.mozilla.org/~jhirsch/universal-search-addon/*"
+    ],
+    "UPDATES": [
```

Comment:
Seems like the long-term source for this info will be the django server. This is fine for now, but definitely file a bug to replace it with a server ping.